src/openzeppelin/account/interface.cairo

```diff
@@ -12,10 +13,12 @@ struct Call {
     calldata: Array<felt252>
 }
 
-trait IAccount {
+trait IStandardAccount {
```

Comment:
Hmm should the trait be `ISRC6` given the forthcoming SNIP?

Reply:
Good point, I think it make sense yes.